src/webapi/node.rs

```diff
@@ -362,6 +362,48 @@ pub struct Node( Reference );
 impl IEventTarget for Node {}
 impl INode for Node {}
 
+impl Node {
+    /// Attempt to create the `Node` from raw html.
+    ///
+    /// # Panics
+    ///
+    /// This method panics if the resulting number of nodes != 1. To ensure this
+    /// it is recommended to at _least_ have programatic control of your root
+    /// HTML node.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let node = Node::from_html("<div>Some text, horray!</div>").unwrap();
+    /// ```
+    pub fn from_html(html: &str) -> Result<Node, SyntaxError> {
+        let result: Result<Value, Value> = js_try!(
```

Comment:
The `js_try!` can convert to the necessary types by itself, so you can get rid of `let: Result<Value, Value>`.